src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -747,6 +784,40 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
                 || WeightableLayerTransformation::isAsymmetricOnWeights(node, defaultPrecisions);
         });
 
+        lptPassConfig->set_callback<TransposeTransformation>([&](const_node_ptr& node) -> bool {
```

Comment:
[random spot] Could you please try to add a few tests for the whole pipeline? We can probably reuse infra for the transformations tests, and run full pipeline instead of a single pass. Would be good to check 3 cases: Compression only, Quantization only and Hybrid case.

Reply:
Applied

Reply:
You've added a functional test which doesn't guarantee that transformations work as expected. I.e. Perf issues due to not applying LPT/compression handling won't be detected. Could you add unit tests similar to tests here https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_gpu/tests/unit/transformations ? Can be done in separate PR if you want

Reply:
Thanks comment.
For the clarification, one question, You mentioned, add unit tests similar in https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_gpu/tests/unit/transformations, and it looked for testing pass itself.
What you want is to add test codes to all the passes(MarkDequantizationSubgraph, TransposeTransformation, MarkupPrecisions, FoldConvertTransformation, FuseConvertTransformation) used in the PR, using WC FC and quantize convolution as inputs?
I just added test code in subgraph which used in problem model.(sd1.5 hybrid)

Reply:
@hyunback Not exactly. My ask is to add unit test for TransoformationPipeline class. I.e. you create an input model, run full pipeline, compare with expected model